### PR TITLE
[Release] Python Data Wrangling & Analytics

### PR DESCRIPTION
**Pull request by COMPOSTCOMEFORTH**
https://www.notion.so/Learning-Python-Jupyter-Notebook-7dd692d0a30f4cc79d30a5f2f21b07f4
https://www.notion.so/Learning-Pandas-Numpy-f313fe979648408fb9d3bff6e46595c0
https://www.notion.so/Setup-Jarvis-PSQL-Data-Warehouse-598a3724b93a4e3fa7691651c2b7fa57
https://www.notion.so/Data-Wrangling-Analytics-39edfdb312ef414f86bdfd4e2713a533
https://www.notion.so/README-md-1d9bd311866543e881f9d4cc39bc45eb

### CODE REVIEW

File: python_data_wrangling/.config/configstore/update-notifier-npm.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"optOut": false,
```
Comment (jarviscanada) [inline python_data_wrangling/.config/configstore/update-notifier-npm.json]: git rm  python_data_wrangling/.cache

Comment (jarviscanada) [reply python_data_wrangling/.config/configstore/update-notifier-npm.json]: please `git rm` all hidden files/dirs

**Review by jarviscanada — APPROVED** [review]

File: python_data_wrangling/.config/configstore/update-notifier-npm.json (hunk 1 above)
Comment (jarviscanada) [inline python_data_wrangling/.config/configstore/update-notifier-npm.json]: git rm  python_data_wrangling/.assets

File: python_data_wrangling/.ipynb_checkpoints/Ch01 Using pandas-checkpoint.ipynb [hunk 2]
```diff
@@ -0,0 +1,150 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
```
Comment (jarviscanada) [inline python_data_wrangling/.ipynb_checkpoints/Ch01 Using pandas-checkpoint.ipynb]: git rm  python_data_wrangling/.ipynb_checkpoints

File: python_data_wrangling/.ipython/profile_default/startup/README [hunk 3]
```diff
@@ -0,0 +1,11 @@
+This is the IPython startup directory
+
+.py and .ipy files in this directory will be run *prior* to any code or files specified
+via the exec_lines or exec_files configurables whenever you load this profile.
+
+Files will be run in lexicographical order, so you can control the execution order of files
+with a prefix, e.g.::
```
Comment (jarviscanada) [inline python_data_wrangling/.ipython/profile_default/startup/README]: remove all hidden dirs

File: python_data_wrangling/psql/data_explore.sql [hunk 4]
```diff
@@ -0,0 +1,27 @@
+-- Show Table Schema
+\d+ retail;
+
+-- Show First 10 Rows
+SELECT * FROM retail limit 10;
+
+-- Number of Records
+SELECT COUNT(*) FROM retail;
```
Comment (jarviscanada) [inline python_data_wrangling/psql/data_explore.sql:8]: this is not complete

File: python_data_wrangling/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+Of particular note are the results of the RFM segmentation found at the bottom of the notebook. In
+particular, there are three segments which deserve attention. The "Can't Lose" segment contains
+customers who previously made large purchases, but have since stopped. In order to regain their
+patronage, a targeted campaign should be ran for them, making specific use of their previously
+purchased products, while also including new ones similar to them. The "Hibernating" segment is
+similar to the "Can't Lose" segment, in that it includes customers who purchased in the past, but
+were not major contributors to the revenue. A campaign similar to the one ran for the "Can't Lose"
+segment can be ran to entice them into becoming more frequent customers. Finally, the "Champions"
+segment contains repeat customers who are responsible for a large portion of the revenue. While not
+at as much of a risk of leaving as the "Can't Lose" segment, it would still be prudent to run a
```
Comment (jarviscanada) [inline python_data_wrangling/README.md:38]: update your profile.yaml

**Review by jarviscanada — CHANGES_REQUESTED** [review]
please fix and submit another review request

File: python_data_wrangling/.config/configstore/update-notifier-npm.json (hunk 1 above)
Comment (COMPOSTCOMEFORTH) [reply python_data_wrangling/.config/configstore/update-notifier-npm.json]: Wouldn't removing the .assets folder break the README images?

**Review by jarviscanada — APPROVED** [review]
LGTM

**Pull request merged**